app/src/main/res/layout/activity_settings.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.activities.SettingsActivity">
+
+    <TextView
```

Comment:
Can you make these "TextView" as ["android.support.v7.widget.AppCompatTextView"](https://developer.android.com/reference/android/support/v7/widget/AppCompatTextView)?

Reply:
All of the TextViews on this file ^^

Reply:
Also add a click effect to each TextView, because these are supposed to work like buttons, right?

Reply:
It will work only by adding clickListener to textView which I have already added :+1: